qhana_plugin_registry/tasks/url_mapped_requests.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2023 QHAna plugin runner contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Functions for opening files from external URLs."""
+
+from re import Pattern
+
+from flask import Flask
+from flask.globals import current_app
+from requests import Session
+from requests.models import Response
+
+REQUEST_SESSION = Session()
+
+
+def map_url(url: str, config_key: str) -> str:
```

Comment:
Maybe provide type hint with a union of literal strings to avoid typos in the config keys (if possible with the current python version)